tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc#basic says "basic but no in-depth knowledge of Perl is needed".